guide/english/certifications/javascript-algorithms-and-data-structures/basic-javascript/nesting-for-loops/index.md

```diff
@@ -93,9 +93,4 @@ multiplyAll([[1,2],[3,4],[5,6,7]]);
 <li>:warning: <strong>DO NOT</strong> add solutions that are similar to any existing solutions. If you think it is similar but better, then try to merge (or replace) the existing similar solution.</li>
```

Comment:
I think this issue should be handled on a separate PR which captures all such occurrences instead of just the ones on these 58 files, since it was not something I changed or caused on this PR.

It is kind of like if you found typos in other sections of these files.  That would be beyond this PR's scope.

On a related note, I created [an issue](https://github.com/freeCodeCamp/freeCodeCamp/issues/35453) a while back for the sole purpose of coming up with what the MVP of a curriculum-related Guide article should even look like.  I personally want to simplify all of these and move away from all of these icons/images.

Also, I have created [another issue](https://github.com/freeCodeCamp/freeCodeCamp/issues/35479) (and PR) related to removing all the repl.it and codepen.io links from all of the curriculum-related Guide articles.  Now that we are getting the number of open Guide articles down, we can start doing these kind of mass-change PRs to clean up these articles.

Reply:
Yeah, I suspected as much.

I saw those issues and am all in favour.  In the mean time, I'll see about creating a PR for these bugs.  Even though the emojis are not long for this world, it's an easy fix. It's only these 3 files as far as I can see.

Reply:
@ojeytonwilliams FYI - you are free to make the changes regarding emoji fixes on this PR if you want.  I just did not want to hold it up for that.  You can add another commit which does this.  I have no issues with that.

Reply:
@RandellDawson No, no it's fine - I'll create a separate PR.  You're right that it's not really connected, it just happens in the same files.

Also, I'm about to sign off for the day, so I'd only end up needlessly delaying this PR.

Reply:
OK.  I think @thecodingaviator is going to merge it soon.